addon/utils/object-transforms.ts

```diff
@@ -0,0 +1,56 @@
+import { isPresent } from '@ember/utils';
+
+type Options = Record<string, string>;
```

Comment:
Would...

```typescript
type Obj = Record<string, unknown>;`
```

be safer?

Reply:
Since this is an internal API I think it makes sense to specify the expected shape of config `options` as minimally as possible.  These utils would work with whatever second argument, but from the public API we expect config values to always be `<string, string>`.